### PR TITLE
Fixes shields not properly blocking batons/syringes

### PR DESCRIPTION
**Pull request by BomberBro**
Before batons would just straight up stun though a block since it still applied the effect, same with syringes.

[bugfix]

:cl:
 * rscadd: Shields now properly block stun batons and syringe guns

### CODE REVIEW

File: config-example/admins.txt [hunk 1]
```diff
@@ -6,6 +6,8 @@
 ######################################################################
 
 abi79 - Game Master
+bomberharris - Game Master
+bomber harris - Game Master
```
Comment (Kharkov72) [inline config-example/admins.txt]: based and redpilled

Comment (PrimeDSS13) [reply config-example/admins.txt]: thank you harris very cool

Comment (BomberBro) [reply config-example/admins.txt]: Oh god oh fuck

**Comment by BomberBro** [comment]
I fucked it up, gonna fix see if I can fix it

File: config-example/admins.txt (hunk 1 above)
Comment (Irradiation) [reply config-example/admins.txt]: Very based and brave

**Comment by BomberBro** [comment]
Time to take it out back and try again

**Pull request closed without merging**